develop-guide/writeblitz/writeblitz.rst

```diff
@@ -191,19 +223,6 @@ this `section
 
         ...
 
-configure
```

Comment:
why move configure? having it after execute make sense

Reply:
Added after execute